### PR TITLE
[Do Not Merge] Triage Tool - Two factor changes

### PR DESCRIPTION
**Pull request by siva-muruganandham**

### CODE REVIEW

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    url: apiCall.url,
+                    contentType: "application/json",
+                    dataType: "json",
+                    success: function (data) {
+                        if (data && data.triageLevelTwo) {
+                            levelTwoItems = data.triageLevelTwo
+                        }
+                        var optionHtml = updateOptionList(levelTwoItems, selectedLevelOne);
+                        levelTwo.html(optionHtml);
+
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:30]: please remove this line

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    },
+                    failure: function () {
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:32]: failure and error should be equivalent.

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                    error: function () {
+                        console.log('Error, Retrieving Triage Tool two options');
+                    }
+                });
+            }
+        } else {
+            var optionHtml = updateOptionList(levelTwoItems, selectedLevelOne);
+            levelTwo.html(optionHtml);
+            levelTwo.attr('disabled', 'disabled');
+
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:44]: please remove this line

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        }
+    });
+
+    function updateOptionList(levelTwoItems, selectedLevelOne) {
+        var optionHtml = '';
+
+        $.each(levelTwoItems,
+            function (index, levelTwo) {
+                if (levelTwo && levelTwo.levelOne && selectedLevelOne === levelTwo.levelOne.title) {
+
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:54]: please remove this line

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+                    optionHtml = optionHtml + generateOptionHtml(levelTwo.title, levelTwo.title);
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:55]: can change to +=

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+                }
+            });
+        return optionHtml;
+    }
+    function generateOptionHtml(value, item)
+    {
+            return '<option value="' + value + '">' + item + '</option>';
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:62]: please change alignment.

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:64]: please remove empty lines

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
+$(document).ready(function () {
+
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:2]: please remove this line

File: src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss [hunk 9]
```diff
@@ -15,6 +15,57 @@ div.dfc-app-pages-alignment-justify {
...
+        font-family: GDS Transport Website;
+        font-size: 19px;
+        font-weight: 700;
+        line-height: 24px;
+        text-align: left;
+        color: #0B0C0C
+    }
+
+    .dfc-app-pages-triage-sub-header {
+        //styleName: Paragraph/Body;
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss:57]: please remove unused

File: src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss [hunk 10]
```diff
@@ -15,6 +15,57 @@ div.dfc-app-pages-alignment-justify {
     text-justify: inter-word;
 }
 
+
+div.dfc-app-pages-triage-panel {
+    // width: 960px;
```
Comment (ZhaomingLu2009) [inline src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss:20]: please remove unused line

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js (hunk 1 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js]: done

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js (hunk 2 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js:50]: done

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js (hunk 3 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js]: fixed

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js (hunk 4 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js]: fixed

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js (hunk 5 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js]: fixed

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js (hunk 6 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js]: fixed

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js (hunk 7 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js]: fixed

File: src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js (hunk 8 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/js/compui.bundles/dfc-app-pages/two-factor-selector.js]: fixed

File: src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss (hunk 9 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss]: fixed

File: src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss (hunk 10 above)
Comment (siva-muruganandham) [reply src/nationalcareers_toolkit/assets/src/frontend/sass/compui/dfc-app-pages.scss]: fixed

**Review by ZhaomingLu2009 — APPROVED** [review]

**Comment by bmatthews-DFE** [comment]
These branches have been moved into the ADO equivalent

**Pull request closed without merging**